packages/payload/src/collections/operations/create.ts

```diff
@@ -128,7 +128,7 @@ async function create<TSlug extends keyof GeneratedTypes['collections']>(
       data,
       overwriteExistingFiles,
       req,
-      throwOnMissingFile: !shouldSaveDraft,
+      throwOnMissingFile: !shouldSaveDraft && collection.config.upload.filesRequiredOnCreate,
```

Comment:
This should be a bit more explicit. Right now, this is written as though you would manually NEED to define `upload.filesRequiredOnCreate: true` for file presence to be enforced.

Rather, we should write something like: 

```
throwOnMissingFile: !shouldSaveDraft && collection.config.upload.filesRequiredOnCreate !== false,
```

Reply:
All done Jmi, all done.